src/components/Input/__tests__/Input.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, RenderResult, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { Input } from '../Input';
+
+describe('<Input> that renders an input field', () => {
+    let view: RenderResult;
+
+    it('should render default input correctly', () => {
+        const onChange = jest.fn();
```

Comment:
I've added onChange here to fix 
![image](https://github.com/textkernel/oneui/assets/104063210/368db56f-7196-41c5-af32-df6938c0fea3)